<launch>  
  <param name="locations_x" value="19.28,19.25,10.28,10.31,14.41,16.96,18.22" /> 
  <param name="locations_y" value="29.12,27.61,28.18,25.86,24.78,22.40,24.92" />
  <param name="goal_tolerance" value="0.5" />
  <param name="pause_time" value="2.0" />
  <param name="log_file" value="$(find patrol_bot)/patrol_log" />

  <param name="/use_sim_time" value="true"/>
  <include file="$(find patrol_bot)/move_base_config/move_base.xml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(find patrol_bot)/stage_config/maps/willow-full-0.05.pgm 0.05"/>
  <node pkg="stage_ros" type="stageros" name="stageros" args="$(find patrol_bot)/stage_config/worlds/willow-pr2-5cm.world"/>   
  <include file="$(find patrol_bot)/move_base_config/amcl_node.xml"/> 

  <node name="patrol_bot" pkg="patrol_bot" type="patrol_bot" output="screen"/>
</launch>
